ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -4100,6 +4100,23 @@ AHU_CHWSC_HWSC_ECON_EDBPC_RFSS2X_RFVSC2X_SFSS2X_SFVSC2X_SSPC:
   - SFVSC2X
   - SSPC
 
+AHU_SFVSC_EFVSC_HTWHLSTC_CHWVM_HWVM_SDM_EDM_SFM_FDPSM_CO2C_SDM2X_EDM2X:
+  description: "Multi-zone chilled water and heating water AHU with heat wheel, supply, exhaust fan speed control"
+  is_canonical: true
+  implements:
+  - AHU
+  - SFVSC
+  - EFVSC
+  - HTWHLSTC
```

Comment:
If this a heat wheel that is transferring heat between the supply and exhaust streams, this equipment seems like a DOAS unit instead (where there is no mixed air and the supply/exhaust streams are isolated from each other).

Reply:
okay thank you, double checked and changed to DOAS